libs/libsearpc/Makefile

```diff
@@ -8,7 +8,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libsearpc
-PKG_VERSION:=5.1.4
+PKG_VERSION:=6.0.7
```

Comment:
Yes. As per the build instructions, a tag named "v3.0-latest" needs to be used. The commit represented by this tag hasn't changed since the last packaged version hence the identical commit hash.